README.md

```diff
@@ -78,19 +78,13 @@ docker-compose up
 Run database schema migration as well:
 
 ```bash
-docker-compose exec mds poetry run alembic upgrade head
-```
-
-Create test database:
-
-```bash
-docker-compose exec db createdb -U mds test_mds
+docker-compose exec app alembic upgrade head
```

Comment:
I removed the `createdb` step to automate it. We could also remove the `alembic upgrade head` step since it's already in `docker-compose.yml`, but that seems like handy documentation to have visible.